File: Try_GameEngine5/src/com/example/try_gameengine/framework/IDrawEvent.java
package com.example.try_gameengine.framework;

import android.graphics.Canvas;
import android.graphics.Paint;

public interface IDrawEvent {
	public void drawChessboardLines(Canvas canvas, Paint paint);
	public void drawAllExistPoints(Canvas canvas);
	public void drawPlayerPocessableMovePoints(Canvas canvas);
}

File: Try_GameEngine5/src/com/example/try_gameengine/framework/IDrawEvent.kt
package com.example.try_gameengine.framework

import android.graphics.Canvas
import android.graphics.Paint

interface IDrawEvent {
    fun drawChessboardLines(canvas: Canvas, paint: Paint)
    fun drawAllExistPoints(canvas: Canvas)
    fun drawPlayerPocessableMovePoints(canvas: Canvas)
}
